src/profiler.h

```diff
@@ -29,22 +29,63 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "util/timetaker.h"
 #include "util/numeric.h"      // paging()
 
-// Global profiler
```

Comment:
Global profiler declaration is moved under Profiler class bacause added "Profiling tools" group (see below) introduces global profiler and its usage now.
Profiler pre-definition isn't needed anymore.